runners/portability/java/build.gradle

```diff
@@ -32,19 +32,26 @@ configurations {
 
 dependencies {
   compile library.java.vendored_guava_26_0_jre
-  compile library.java.hamcrest_library
   compile project(":runners:java-fn-execution")
   compile project(":runners:java-job-service")
   compile project(path: ":sdks:java:harness", configuration: "shadow")
   compile library.java.vendored_grpc_1_26_0
   compile library.java.slf4j_api
-
+  compile library.java.joda_time
```

Comment:
nitpick - there should be a blank line between "compile" and "testcompile" lines like before